lib/src/chunking.rs

```diff
@@ -294,20 +311,27 @@ impl Chunking {
             .unwrap();
 
         // TODO: Compute bin packing in a better way
+        let start = Instant::now();
         let packing = basic_packing(
             sizes,
             NonZeroU32::new(self.max).unwrap(),
             prior_build_metadata,
         );
+        let duration = start.elapsed();
+        println!("Time elapsed in packing: {:#?}", duration);
```

Comment:
Let's use something more like `tracing::debug!` for this so it's configurable.  In general, *library* code should not print to stdout unless it's asked I think.